gui/game_list.cpp

```diff
@@ -0,0 +1,100 @@
+#include <algorithm>
+#include <cctype> // For tolower()
+#include <string>
+#include <vector>
```

Comment:
We have our own portable implementations for these

- algorithm -> [Common::Algorithm](https://github.com/scummvm/scummvm/blob/master/common/algorithm.h)
- cctype (tolower) -> Use Common::String.toLowercase()
- string -> [Common::String](https://github.com/scummvm/scummvm/blob/master/common/str.h)
- vector -> [Common::Array](https://github.com/scummvm/scummvm/blob/master/common/array.h)